lib/data/plugins.json

```
@@ -1307,6 +1307,12 @@
     "repository": "https://github.com/hellotoby/metalsmith-to-json",
     "description": "Generates JSON files from markdown, preserving filenames and frontmatter."
   },
+  {
+    "name": "Tweet Embed",
+    "icon": "gridlines",
+    "repository": "https://github.com/scurker/metalsmith-tweet-embed",
+    "description": "Converts Twitter status URLS to embedded Twitter statuses"
+  }
```

Still missing a comma:

```suggestion
  },
```

---

Thanks! Sorry I missed this.